### PR TITLE
Add Dutch prominent words

### LINKED ISSUE #1016: Add prominent words for Dutch
We should add the prominent words analysis for Dutch.

### PR DESCRIPTION
**Pull request by IreneStr**
## Summary

This PR can be summarized in the following changelog entry: Add prominent words for Dutch.

## Relevant technical choices:

* Add Dutch function words.
* Add Dutch to relevantWords.js.
* Add filters for Dutch.

## Test instructions

This PR can be tested by following these steps:

* Use Node to test: node ./examples/wordRelevance/index.js [txt filename] nl_NL

Fixes #1016

### CODE REVIEW

File: js/researches/dutch/functionWords.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+// These words and abbreviations are frequently used in recipes in lists of ingredients.
+let recipeWords = [ "ml", "cl", "dl", "l", "tl", "el", "mg", "g", "gr", "kg", "ca", "theel", "min", "sec", "uur"  ];
+
+let vagueNouns = [ "ding", "dingen", "manier", "manieren", "item", "items", "keer", "maal", "procent", "geval", "aspect", "persoon",
+	"personen", "deel" ];
+
+// 'No' is already included in the quantifier list.
+let miscellaneous = [ "niet", "wel", "ja", "nee", "neen", "oké", "oke", "okee", "ok", "niets", "zoiets", "%", "€", "euro" ];
+
+module.exports = function() {
```
Comment (andizer) [inline js/researches/dutch/functionWords.js]: Can you add some docs?

File: js/researches/english/functionWords.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ var notFilteredPassiveAuxiliaries = require( "./passivevoice/auxiliaries.js" )()
 var transitionWords = require( "./transitionWords.js" )().singleWords;
 
 /**
- * Returns an array with exceptions for the keyword suggestion researcher.
+ * Returns an array with exceptions for the prominent words researcher
```
Comment (andizer) [inline js/researches/english/functionWords.js]: I would have done this in a separate pull.

Comment (IreneStr) [inline js/researches/english/functionWords.js]: Will do the next time!

File: js/researches/dutch/functionWords.js [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+	"er", "foei", "hmm", "nou", "nee", "tja", "nja", "okido", "ho", "halt", "komaan", "komop", "verrek", "nietwaar", "brr", "oef",
+	"ach", "och", "bah", "enfin", "afijn", "haha", "hihi", "hatsjie", "hatsjoe", "hm", "tring", "vroem", "boem", "hopla" ];
+
+// These words and abbreviations are frequently used in recipes in lists of ingredients.
+let recipeWords = [ "ml", "cl", "dl", "l", "tl", "el", "mg", "g", "gr", "kg", "ca", "theel", "min", "sec", "uur"  ];
+
+let vagueNouns = [ "ding", "dingen", "manier", "manieren", "item", "items", "keer", "maal", "procent", "geval", "aspect", "persoon",
+	"personen", "deel" ];
+
+// 'No' is already included in the quantifier list.
```
Comment (andizer) [inline js/researches/dutch/functionWords.js]: I would change `No` into Dutch.

File: js/researches/dutch/functionWords.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+let articles = [ "de", "het", "een", "der", "des", "den" ];
+
+let numerals = [ "eén", "één", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen", "tien", "elf", "twaalf", "dertien",
+	"veertien", "vijftien", "zestien", "zeventien", "achttien", "negentien", "twintig", "eerste", "tweede", "derde", "vierde",
+	"vijfde", "zesde", "zevende", "achtste", "negende", "tiende", "elfde", "twaalfde", "dertiende", "veertiende", "vijftiende",
+	"zestiende", "zeventiende", "achttiende", "negentiende", "twinstigste", "honderd", "honderden", "duizend", "duizenden", "miljoen",
+	"miljoenen", "biljoen", "biljoenen" ];
+
+// 'Het' is already included in the list of articles.
+let personalPronounsNominative = [ "ik", "je", "jij", "hij", "ze", "we", "wij", "jullie", "zij" ];
```
Comment (terw-dan) [inline js/researches/dutch/functionWords.js]: should `gij` be added to this list?

**Comment by terw-dan** [comment]
CR 👍

**Comment by terw-dan** [comment]
Acceptance 👍

**Pull request merged**